Makefile

```diff
@@ -147,3 +151,48 @@ docs/pykanidm/serve:
 	cd pykanidm && \
 	poetry install && \
 	poetry run mkdocs serve
+
+########################################################################
+
+release/kanidm: ## Build the Kanidm CLI
+	cargo build -p kanidm_tools --bin kanidm --release
+
+release/kanidmd: ## Build the Kanidm daemon
```

Comment:
The problem here is this will be missing the release profiles so it'll try to use the relatively placed wasm resources rather than the usr/share bits.

Reply:
Oh and it's missing CPU optimisations.

Reply:
Sorted 😄 As discussed elsewhere, these are intended to be the patterns for packages, with the builders applying distribution/build specific flags in the build pipelines.